blocks/generator_test.go

```diff
@@ -159,13 +159,12 @@ func createModifiedATXs(
 			types.NIPostChallenge{PublishEpoch: lid.GetEpoch()},
 			address,
 			numUnit,
-			nil,
```

Comment:
I would even go so far as to get rid of `types.NewActivationTx` completely. All fields it sets are public, i.e. the caller of that function can just create the `types.ActivationTx` themselves.

Reply:
Yeah, that makes sense absolutely, but I don't won't to stretch this PR with this kind of refactor.